pkg/resources/ops/collector.go

```diff
@@ -20,21 +20,25 @@ import (
 	"fmt"
 
 	"k8s.io/kops/pkg/apis/kops"
+	kopsapi "k8s.io/kops/pkg/apis/kops"
```

Comment:
this is a duplicate import with the line above it

Reply:
fixed!